AutoGear.lua

```diff
@@ -1710,6 +1710,8 @@ SlashCmdList["AutoGear"] = function(msg)
         AutoGearPrint("AutoGear: Looks like you are "..AutoGearGetSpec().."."..((AutoGearDB.UsePawn or AutoGearDB.Override) and ("  However, AutoGear is using "..(AutoGearDB.UsePawn and "Pawn" or "\""..AutoGearDB.OverrideSpec.."\"").." for gear evaluation due to the \""..(AutoGearDB.UsePawn and "use Pawn to evaluate upgrades" or "override specialization").."\" option.") or ""), 0)
     elseif (param1 == "verbosity") or (param1 == "allowedverbosity") then
         SetAllowedVerbosity(param2)
+    elseif (param1 == "setspec") then
```

Comment:
This is a good idea, but will need to be documented in AutoGearPrintHelp.

Reply:
This part is conflicted and no longer relevant.  Please merge my changes and resolve the conflict in your branch.